services/storage/tests/unit/test_s3_client.py

```diff
@@ -56,39 +54,35 @@ def mock_config(
     monkeypatch.setenv("STORAGE_POSTGRES", "null")
 
 
-async def test_storage_storage_s3_client_creation(app_settings: Settings):
+async def test_storage_storage_s3_client_creation(
+    app_settings: Settings,
+):
     assert app_settings.STORAGE_S3
-    async with AsyncExitStack() as exit_stack:
-        storage_s3_client = await StorageS3Client.create(
-            exit_stack,
-            app_settings.STORAGE_S3,
-            app_settings.STORAGE_S3_CLIENT_MAX_TRANSFER_CONCURRENCY,
-        )
-        assert storage_s3_client
-        response = await storage_s3_client.client.list_buckets()
-        assert not response["Buckets"]
-    with pytest.raises(botocore.exceptions.HTTPClientError):
-        await storage_s3_client.client.list_buckets()
+    storage_s3_client = await StorageS3Client.create(
+        app_settings.STORAGE_S3,
+        app_settings.STORAGE_S3_CLIENT_MAX_TRANSFER_CONCURRENCY,
+    )
+    assert storage_s3_client
+    response = await storage_s3_client.client.list_buckets()
+    assert not response["Buckets"]
```

Comment:
the way you changed that test does not anymore properly test that the client is properly closed. I think this needs some changes.